ai-decision-services/ai-ad-dpp-engine/run-tests/run-tests.md

```diff
@@ -0,0 +1,280 @@
+Lab 8: A simple data processing, training and inferencing example
+===
+
+## Training
+
+To make sure you have everything setup well in training, you can upload the training data to the **training-data-bucket**. If you see a Data Flow run is triggered is successful, and under Anomaly Detection a model is trained successfully, then the training path is working.
+
+Fundamentally, you should see processed data in csv under your **staging-bucket**:
+
+![](./images/Exp1.png)
+
+And under your **results-bucket**, your model information will be written to a file named **model_info.** It should look like this:
+
+```java
+{
+    "model_ids": [
+        {
+            "model_id": "ocid1.aianomalydetectionmodel.oc1.phx.amaaaaaaor7l3jiavbwqxnwiqvikmrdjyeaq7e2bvrpbh3y5apq4iv4xmfga",
+            "columns": [
+                "timestamp",
+                "sensor_01",
+                "sensor_02",
+                "sensor_03",
+                "sensor_04",
+                "sensor_05",
+                "sensor_06",
+                "sensor_07",
+                "sensor_08",
+                "sensor_09",
+                "sensor_10",
+                "sensor_11",
+                "sensor_12",
+                "sensor_13"
+            ]
+        }
+    ]
+}
+```
+
+Lastly, you should also see your model trained under **Anomaly Detection**. You can check the FAP and other details here:
```

Comment:
Reword - Review the trained model in **Anomaly Detection** Service.  Check the FAP and other details.  
(Remove the ":" at the end of the sentence. 
Thanks.